### ISSUE TITLE
[SF] Dialog - NVDA does not read out the dialog content

### ISSUE THREAD

**Issue by Vir-Mittal** [SAP/ui5-webcomponents#5350]
### **Bug Description**
Dialog - NVDA does not read out the dialog content, it only reads the title and footer options.

### **Expected Behavior**
The content of the dialog should also be read

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-dialog--default-story
2. Use NVDA
3. Click on the open dialog button

Screenshot - 
<img width="1764" alt="Screen Shot 2022-06-09 at 4 14 07 PM" src="https://user-images.githubusercontent.com/36656306/172960558-7efb048b-db89-4d50-a52a-ad8cba0f85e4.png">

### **Context**
- UI5 Web Components React version: {0.25.0}
- OS/Platform: {Windows}
- Browser: {Chrome}
- Affected component: {Dialog}

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Business impact: {...}

**Comment by niyap** [comment]
Hello @SAP/ui5-webcomponents-topic-rd,

In the sample provided, the dialog content is not read out with NVDA, but read out with JAWS 2021/2022: https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-dialog--default-story
I checked out our samples in Playground and in our test page and the content is not read out both with JAWS and NVDA.
On the other hand, in OpenUI5 it is read out: https://sapui5.hana.ondemand.com/?sap-ui-language=en#/entity/sap.m.Dialog/sample/sap.m.sample.Dialog and there the content is read out.

Could you please check and tell the reported whether it is expected for the content to be read out?

Thank you in advance!

Kind Regards,
Niya
